docs/installation.rst

```diff
@@ -6,135 +6,219 @@ Installationsvejledning
 
 .. code-block::
 
-    (base) C:\FIRE>conda env create --file environment.yml
-    Collecting package metadata (repodata.json): done
-    Solving environment: done
+    (base) C:\FIRE>mamba env create --file environment.yml
+    pkgs/main/win-64                                              No change
+    pkgs/main/noarch                                              No change
+    pkgs/r/win-64                                                 No change
+    pkgs/msys2/win-64                                             No change
+    pkgs/r/noarch                                                 No change
+    pkgs/msys2/noarch                                             No change
+    conda-forge/noarch                                  10.8MB @   3.8MB/s  4.2s
+    conda-forge/win-64                                  18.7MB @   3.2MB/s  8.5s
+
+
+    Looking for: ['click=8.0', 'cx_oracle=8.3', 'fiona=1.8', 'gama=2.15', 'openpyxl=3.0', 'pandas=1.3', 'pyproj=3.3', 'python=3.9', 'sqlalchemy=1.4', 'xmltodict=0.12']
+
+
+    Transaction
+
+    Prefix: C:\Users\B...\AppData\Local\mambaforge\envs\fire
 
-    Downloading and Extracting Packages
-    cryptography-3.4.7   | 705 KB    | ############################################################################ | 100%
-    libcurl-7.76.1       | 292 KB    | ############################################################################ | 100%
+    Updating specs:
 
-    ...
+    - click=8.0
+    - cx_oracle=8.3
+    - fiona=1.8
+    - gama=2.15
+    - openpyxl=3.0
+    - pandas=1.3
+    - pyproj=3.3
+    - python=3.9
+    - sqlalchemy=1.4
+    - xmltodict=0.12
 
-    coverage-5.5         | 285 KB    | ############################################################################ | 100%
-    libzip-1.7.3         | 134 KB    | ############################################################################ | 100%
+
+    Package                           Version  Build               Channel                  Size
+    ------------------------------------------------------------------------------------------------
+    Install:
+    ------------------------------------------------------------------------------------------------
+
+    + attrs                            22.2.0  pyh71513ae_0        conda-forge/noarch       54kB
+    (...)
+    + zstd                              1.5.2  h7755175_4          conda-forge/win-64     Cached
+
+    Summary:
+
+    Install: 106 packages
+
+    Total download: 121MB
+
+    ------------------------------------------------------------------------------------------------
+
+    python_abi                                           6.1kB @ 106.3kB/s  0.1s
+    (...)
+    libspatialite                                       12.0MB @   1.8MB/s  2.3s
     Preparing transaction: done
     Verifying transaction: done
     Executing transaction: done
     #
     # To activate this environment, use
     #
-    #     $ conda activate fire
+    #     $ conda activate foo
```

Comment:
Hvorfor `foo`?

Reply:
`foo` er mit navn til dét miljø, jeg oprettede for at køre kommandoerne igennem uden at starte forfra med mit eksisterende. Jeg checker lige andre stede efter `foo` og erstatter med `fire`.